ql/pom.xml

```diff
@@ -164,6 +164,15 @@
       <artifactId>hive-storage-api</artifactId>
     </dependency>
     <!-- inter-project -->
+    <dependency>
+      <groupId>com.amazonaws</groupId>
+      <artifactId>aws-java-sdk-bundle</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>com.amazonaws.secretsmanager</groupId>
+      <artifactId>aws-secretsmanager-caching-java</artifactId>
```

Comment:
again a compile time dependency which gets included into any uber jars we build.